gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -96,6 +96,9 @@ type provisionerConfig struct {
 
 	// Optional: Chap Auth Enable
 	chapAuthEnabled bool
+
+	//Optional: Chap Auth Secret Namespace.
+	chapSecretNamespace string
```

Comment:
doc needs update

Reply:
true.. I was planning to hide it :).  any way I will update the doc in this PR.